.github/workflows/update-protobufs.yaml

```diff
@@ -0,0 +1,33 @@
+name: Update Protobufs
+on:
+  push:
+    branches:
+    - yaakov/auto-protos # temporary trigger during development
+  #schedule:
+  #- cron: '0 17 * * SUN'
+
+jobs:
+  update-protobufs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+
+    - name: Run Update Script
+      shell: pwsh
+      run: |
+        git submodule init
+        Resources/ProtobufGen/update.ps1
+
+    # TODO: We need a new GitHub Machine Account (or maybe an existing one?) to generate a PAT that we can use as the token here
+    # otherwise, our new changes will not trigger further Actions (on:push or on:pull_request, i.e. CI/CD builds).
+    #- name: Create Pull Request
+    #  uses: peter-evans/create-pull-request@v3.10.1
+    #  with:
+    #      #token: ${{ secrets.PAT }}
+    #      commit-message: Update protobufs
+    #      title: Update protobufs
+    #      body: |
+    #        - Update protobufs
+    #      branch: auto/protobufs
```

Comment:
Configure `branch-suffix` as well? Not sure what happens if the branch already exists and this action tries to create it again.